package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "@eslint/js": "^9.22.0",
     "@playwright/test": "^1.49.1",
+    "@tsconfig/node22": "^22.0.1",
```

Comment:
let's remove `nodemon` dep if we've stopped using it 👍

Reply:
Removed 🤘🏻